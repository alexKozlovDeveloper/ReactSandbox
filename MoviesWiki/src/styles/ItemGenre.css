.genre {
    color: white;
    padding: 3px;
}