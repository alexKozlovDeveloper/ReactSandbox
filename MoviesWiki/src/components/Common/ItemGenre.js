import React, { Component } from "react";

import styles from "../../styles/ItemGenre.css"

function ItemGenre(props) {
    return  <div className={styles.genre}>
                {props.genre}
            </div>;
}

export default ItemGenre;