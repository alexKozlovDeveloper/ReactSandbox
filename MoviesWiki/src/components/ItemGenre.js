import React, { Component } from "react";

import styles from "../styles/ItemGenre.css"

class ItemGenre extends Component {
    render() {
        return (
            <div className={styles.genre}>
                <div>Genre {this.props.genre}</div>
            </div>
        );
    }
}

export default ItemGenre;