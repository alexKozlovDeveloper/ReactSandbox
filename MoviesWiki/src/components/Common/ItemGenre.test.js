import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import ItemGenre from "./ItemGenre";

configure({adapter: new Adapter()});

test('ItemGenre rendering test', () => {

    var genre = "someGenre";

    const output = shallow(<ItemGenre genre={genre} />);
    expect(shallowToJson(output)).toMatchSnapshot();
 });

