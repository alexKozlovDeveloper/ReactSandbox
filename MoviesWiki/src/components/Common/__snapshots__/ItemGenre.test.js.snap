// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemGenre rendering test 1`] = `
<div
  className="genre"
>
  someGenre
</div>
`;
